### PR TITLE
fix(dropdown): margin missing when the position of floating menu of dropdown is upward

### PR DESCRIPTION
**Pull request by dutrieux**
## Description

When the position of floating menu of dropdown is upward, the margin is missing

## Screenshot (when possible)
**Before**
dropdown

**after**
dropdown

### CODE REVIEW

File: src/definitions/modules/dropdown.less [hunk 1]
```diff
@@ -1299,9 +1299,14 @@ select.ui.dropdown {
   border-radius: @floatingMenuBorderRadius !important;
 }
 .ui.floating.dropdown > .menu {
-  margin-top: @floatingMenuDistance !important;
   border-radius: @floatingMenuBorderRadius !important;
 }
+.ui:not(.upward).floating.dropdown > .menu {
+  margin-top: @floatingMenuDistance !important;
```
Comment (lubber-de) [inline src/definitions/modules/dropdown.less]: Remove the !important, it's not needed

File: src/definitions/modules/dropdown.less [hunk 2]
```diff
@@ -1299,9 +1299,14 @@ select.ui.dropdown {
...
+}
+.ui.upward.floating.dropdown > .menu {
+  margin-bottom: @floatingMenuDistance !important;
```
Comment (lubber-de) [inline src/definitions/modules/dropdown.less]: Also here: !important is not needed

File: src/definitions/modules/dropdown.less (hunk 1 above)
Comment (dutrieux) [reply src/definitions/modules/dropdown.less]: That's how it is on version 2.7.4 : sure you want I change that ?

Comment (lubber-de) [reply src/definitions/modules/dropdown.less]: Yes, i tested it before posting and the !important is still inherit from SUI, not a specific 2.7.4 feature.
But of course, i advise you to test it and get convinced yourself.

**Pull request merged**